test/games/prime1/exporter/test_prime_game_exporter.py

```diff
@@ -5,17 +5,26 @@
 import pytest
 
 from randovania.games.prime1.exporter.game_exporter import PrimeGameExporter, PrimeGameExportParams, adjust_model_names
+from randovania.games.prime1.layout.prime_configuration import EnemyAttributeRandomizer
 
 
 @pytest.mark.parametrize('use_echoes_models', [True, False])
-def test_patch_game(mocker, tmp_path, use_echoes_models):
+@pytest.mark.parametrize('use_enemy_attribute_randomizer', [False, True])
+def test_patch_game(mocker, tmp_path, use_echoes_models, use_enemy_attribute_randomizer):
     mock_symbols_for_file: MagicMock = mocker.patch("py_randomprime.symbols_for_file", return_value={
         "UpdateHintState__13CStateManagerFf": 0x80044D38,
     })
+    if use_enemy_attribute_randomizer:
+        use_enemy_attribute_randomizer = EnemyAttributeRandomizer(1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, False).as_json
+    else:
+        use_enemy_attribute_randomizer = None
+
     mock_patch_iso_raw: MagicMock = mocker.patch("py_randomprime.patch_iso_raw")
     mock_asset_convert: MagicMock = mocker.patch("randovania.patching.prime.asset_conversion.convert_prime2_pickups")
+    mock_enemy_data: MagicMock = mocker.patch("Random_Enemy_Attributes.Random_Enemy_Attributes.PyRandom_Enemy_Attributes")
     mocker.patch("randovania.games.prime1.exporter.game_exporter.adjust_model_names")
-    patch_data = {"patch": "data", 'gameConfig': {}, 'hasSpoiler': True, "preferences": {}, "roomRandoMode": "None"}
+    patch_data = {"patch": "data", 'gameConfig': {}, 'hasSpoiler': True, "preferences": {}, "roomRandoMode": "None",
+                  "randEnemyAttributes": use_enemy_attribute_randomizer, "seed": 103817502} # I still have the "seed" here because if I don't have this here, my function creates a  KeyError: 'seed'
```

Comment:
you should store the seed number in a variable.

What you're doing wrong with the seed is that your code isn't poping it from the config, like you do with `"randEnemyAttributes"`. So it stays into what is sent to randomprime. You either should pop it so it's not sent, or update  the `expected` dict below to properly expect this value.